code/android-core-library/api/android-core-library.api

```diff
@@ -0,0 +1,1075 @@
+	public fun equals (Ljava/lang/Object;)Z
+	public final fun getDetail ()Ljava/util/Map;
+	public final fun getId ()Ljava/lang/String;
+	public final fun getType ()Ljava/lang/String;
+	public fun hashCode ()I
+	public final fun setDetail (Ljava/util/Map;)V
+	public fun toString ()Ljava/lang/String;
+}
+
+public class com/adobe/marketing/mobile/launch/rulesengine/download/RulesLoadResult {
+	public fun <init> (Ljava/lang/String;Lcom/adobe/marketing/mobile/launch/rulesengine/download/RulesLoadResult$Reason;)V
+	public fun getData ()Ljava/lang/String;
+	public fun getReason ()Lcom/adobe/marketing/mobile/launch/rulesengine/download/RulesLoadResult$Reason;
+}
+
+public final class com/adobe/marketing/mobile/launch/rulesengine/download/RulesLoadResult$Reason : java/lang/Enum {
+	public static final field CANNOT_CREATE_TEMP_DIR Lcom/adobe/marketing/mobile/launch/rulesengine/download/RulesLoadResult$Reason;
+	public static final field CANNOT_STORE_IN_TEMP_DIR Lcom/adobe/marketing/mobile/launch/rulesengine/download/RulesLoadResult$Reason;
+	public static final field INVALID_SOURCE Lcom/adobe/marketing/mobile/launch/rulesengine/download/RulesLoadResult$Reason;
+	public static final field NOT_MODIFIED Lcom/adobe/marketing/mobile/launch/rulesengine/download/RulesLoadResult$Reason;
+	public static final field NO_DATA Lcom/adobe/marketing/mobile/launch/rulesengine/download/RulesLoadResult$Reason;
+	public static final field SUCCESS Lcom/adobe/marketing/mobile/launch/rulesengine/download/RulesLoadResult$Reason;
+	public static final field ZIP_EXTRACTION_FAILED Lcom/adobe/marketing/mobile/launch/rulesengine/download/RulesLoadResult$Reason;
+	public static fun valueOf (Ljava/lang/String;)Lcom/adobe/marketing/mobile/launch/rulesengine/download/RulesLoadResult$Reason;
+	public static fun values ()[Lcom/adobe/marketing/mobile/launch/rulesengine/download/RulesLoadResult$Reason;
+}
+
+public class com/adobe/marketing/mobile/launch/rulesengine/download/RulesLoader {
+	public fun <init> (Ljava/lang/String;)V
+	public fun getCacheName ()Ljava/lang/String;
+	public fun loadFromAsset (Ljava/lang/String;)Lcom/adobe/marketing/mobile/launch/rulesengine/download/RulesLoadResult;
+	public fun loadFromCache (Ljava/lang/String;)Lcom/adobe/marketing/mobile/launch/rulesengine/download/RulesLoadResult;
+	public fun loadFromUrl (Ljava/lang/String;Lcom/adobe/marketing/mobile/AdobeCallback;)V
+}
+
+public final class com/adobe/marketing/mobile/launch/rulesengine/json/JSONRulesParser {
+	public static final field INSTANCE Lcom/adobe/marketing/mobile/launch/rulesengine/json/JSONRulesParser;
+	public static final fun parse (Ljava/lang/String;Lcom/adobe/marketing/mobile/ExtensionApi;)Ljava/util/List;
+}
+
+public class com/adobe/marketing/mobile/rulesengine/ComparisonExpression : com/adobe/marketing/mobile/rulesengine/Evaluable {
```

Comment:
The classes under `com/adobe/marketing/mobile/rulesengine` package could be removed.

Reply:
Let us move into internal subpackage if we don't wish to make it public.